src/libraries/System.Reflection.Metadata/tests/Metadata/Ecma335/Encoding/ControlFlowBuilderTests.cs

```diff
@@ -418,5 +418,34 @@ public void Branch_LongInstruction_LongDistance()
                 (byte)ILOpCode.Ret
             }, builder.ToArray());
         }
+
+        [Fact]
+        public void Clear()
+        {
+            var cfb = new ControlFlowBuilder();
+
+            var il1 = GenerateSampleIL(cfb);
+            cfb.Clear();
+            var il2 = GenerateSampleIL(cfb);
+
+            AssertEx.Equal(il1, il2);
+
+            static byte[] GenerateSampleIL(ControlFlowBuilder cfb)
+            {
+                var code = new BlobBuilder();
+                var il = new InstructionEncoder(code, cfb);
+
+                var label = il.DefineLabel();
+
+                il.MarkLabel(label);
+                il.OpCode(ILOpCode.Nop);
+                il.Branch(ILOpCode.Br_s, label);
```

Comment:
Could add an `il` for `_lazyExceptionHandlers` too? To make sure it has always been cleared with the API

Reply:
Done.